src/styles/index.css

```diff
@@ -0,0 +1,161 @@
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: auto;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+  background-image: url(../images/imac.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__title {
+  padding: 0 16px;
+  margin-top: 40px;
+  line-height: 18px;
+  height: 36px;
+  width: 166px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__product-code {
+  box-sizing: border-box;
+  margin-top: 4px;
+  padding: 0 16px;
+  width: 100%;
+  justify-content: center;
```

Comment:
The `justify-content: center;` property is used here, but since the element is not a flex container, this property will have no effect. Consider removing it or ensuring the parent is a flex container.